crates/consensus/Cargo.toml

```diff
@@ -26,4 +29,5 @@ arbitrary = { workspace = true, features = ["derive"] }
 
 [features]
 k256 = ["alloy-primitives/k256", "alloy-network/k256"]
+kzg = ["c-kzg", "sha2", "thiserror"]
```

Comment:
this should be default

Reply:
Please also use `dep:` syntax

Reply:
note that making kzg default breaks the wasm check, and don't think we want to skip the check for consensus. should we have a diff cargo hack check for consensus only with no default features?

Reply:
if we limit kzg to validation only then we don't need this as default